src/molecules/Input/input.scss

```diff
@@ -0,0 +1,9 @@
+.message {
+  font-family: Open Sans;
```

Comment:
open sans? That seems wrong, let's check with Joanne if we should be using Mulish or Lato here, because this would be the ONLY place we use open sans - seems like a hefty import cost for an unimportant piece of UI